test/unit/spec/util/jobLogViewerSpec.js

```diff
@@ -277,7 +281,7 @@ define([
             });
 
             JobsData.example.JobState.invalid.forEach((state) => {
-                describe(`should be awaiting job data with dodgy state ${JSON.stringify(
+                xdescribe(`should be awaiting job data with dodgy state ${JSON.stringify(
```

Comment:
Should this still be x'd out?

Reply:
Similar to the other one -- the job manager / job comm channel should stop bad messages getting out into the frontend.